piqe_ocp_lib/api/tasks/operator_ops.py

```diff
@@ -77,14 +78,26 @@ def add_operator_to_cluster(self, operator_name, target_namespaces=[]):
 
         return True
 
+    def is_operator_installed(self, operator_name, operator_namespace):
+        """
+        Check if operator is installed and returned true or false
+        :param operator_name: name of the operator.
+        :param operator_namespace: namespace of the operator
+        rturn: object of values of csv
+        """
+        csv = self.csv
```

Comment:
Nitpick, this is redundant, you can just do 'csv_resp = self.csv.get_cluster_service_version() in the next line